src/wrapped/wrappedlibjpeg.c

```diff
@@ -309,6 +352,13 @@ EXPORT void* my_jpeg_std_error(x86emu_t* emu, void* errmgr)
 
     wrapErrorMgr(my_lib->w.bridge, ret);
 
+    my->jpeg_std_error(&native_err_mgr);
+    ret->error_exit = (void*)AddBridge(my_lib->w.bridge, vFp, native_error_exit, 0, NULL);
```

Comment:
it's probably better to use `AddCheckBridge` here, to avoid adding multiple time the same value?